Add Biotime data, closes #116

gmyenni opened the pull request.

gmyenni committed:
Add Biotime data, closes #116

ha0ye committed:
fix paths and eliminate redundancies in biotime_datasets_plan
reduce size of test data
add test for biotime dataset plan

ha0ye commented:
@gmyenni I've done some minor cleanup.

Right now, it looks like `get_biotime_data()` loads in all of biotime each time you want a dataset, which might be rather slow. I think we can do the same thing as with BBS, and prep it by subsetting into separate files based on dataset/study_id.

ha0ye committed:
include test data in package
reorganize filenames for data checking functions
update data checking functions

ha0ye committed:
fix #135
update GPDD date for correct date format
